### PR TITLE
Add Keebcats Denis PCB

### PR DESCRIPTION
**Pull request by mini-ninja-64**
This PR adds the Denis 60% PCB

## Description

Currently the picture in the readme is missing, I will add this as soon as production units arrive :)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cutie_club/keebcats/denis/rules.mk [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
```
Comment (drashna) [inline keyboards/cutie_club/keebcats/denis/rules.mk:3]:
```suggestion

# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/cutie_club/keebcats/denis/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Denis
+
+* Keyboard Maintainer: [Cutie Club](https://github.com/cutie-club/)
+* Hardware Supported: Atmega32u4 based 60% PCB, with wide compatibility
+* Hardware Availability: [Keebcats](https://keebcats.co.uk)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/cutie_club/keebcats/denis/readme.md:7]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/cutie_club/keebcats/denis/rules.mk [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/cutie_club/keebcats/denis/rules.mk:8]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/cutie_club/keebcats/denis/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/cutie_club/keebcats/denis/rules.mk:19]:
```suggestion
```

File: keyboards/cutie_club/keebcats/denis/rules.mk [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+AUDIO_ENABLE = no           # Audio output
+LAYOUTS = 60_iso_tsangan 60_iso_split_bs_rshift 60_iso 60_tsangan_hhkb 60_hhkb 60_ansi_tsangan 60_ansi_split_bs_rshift 60_ansi
```
Comment (drashna) [inline keyboards/cutie_club/keebcats/denis/rules.mk:21]:
```suggestion

LAYOUTS = 60_iso_tsangan 60_iso_split_bs_rshift 60_iso 60_tsangan_hhkb 60_hhkb 60_ansi_tsangan 60_ansi_split_bs_rshift 60_ansi
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by mini-ninja-64** [comment]
@drashna thank you so much for the changes, I have made them now :)

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**